tests/base.py

```diff
@@ -99,6 +99,11 @@ def shutdown(self):
 class BaseTest(object):
     def setup_method(self, test_method):
         assert settings.TESTING, "settings.TESTING is False, try `SNUBA_SETTINGS=test` or `make test`"
```

Comment:
We should drop the test table before all tests, not events only.
Could you please bring the logic that drops the table to the parent class ?
Line 118 to line 125
and remove the comment at line 116?

Reply:
Also, BaseTest should provide the tear_down method that drops the table (line 127)

Reply:
Updated